tests/e2e/backup_restore_suite_test.go

```diff
@@ -176,13 +176,13 @@ var _ = Describe("AWS backup restore tests", func() {
 			PreBackupVerify:      mysqlReady,
 			PostRestoreVerify:    mysqlReady,
 		}, nil),
-		Entry("Parks application <4.8.0", BackupRestoreCase{
-			ApplicationTemplate:  "./sample-applications/parks-app/manifest.yaml",
-			ApplicationNamespace: "parks-app",
-			Name:                 "parks-e2e",
+		Entry("Mongo application <4.8.0", BackupRestoreCase{
```

Comment:
Since `Mongo application <4.8.0` is no longer using a different manifests, we can remove the other one.

Reply:
No longer different to `Mongo application >=4.8.0`.